packages/components/src/angle-picker/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef } from '@wordpress/element';
+import { useInstanceId, __experimentalUseDragging as useDragging } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from '../base-control';
+
+function getAngle( centerX, centerY, pointX, pointY ) {
+	const y = pointY - centerY;
+	const x = pointX - centerX;
+
+	const angleInRadians = Math.atan2( y, x );
+	const angleInDeg = Math.round( angleInRadians * ( 180 / Math.PI ) ) + 90;
+	if ( angleInDeg < 0 ) {
+		return 360 + angleInDeg;
+	}
+	return angleInDeg;
+}
+
+const AngleCircle = ( { value, onChange } ) => {
+	const angleCircleRef = useRef();
+	const angleCircleCenter = useRef();
+
+	const setAngleCircleCenter = () => {
+		const rect = angleCircleRef.current.getBoundingClientRect();
+		angleCircleCenter.current = {
+			x: rect.x + ( rect.width / 2 ),
+			y: rect.y + ( rect.height / 2 ),
+		};
+	};
+
+	const changeAngleToPosition = ( event ) => {
+		const { x: centerX, y: centerY } = angleCircleCenter.current;
+		onChange( getAngle( centerX, centerY, event.clientX, event.clientY ) );
+	};
+
+	const { startDrag, isDragging } = useDragging( {
+		onDragStart: ( event ) => {
+			setAngleCircleCenter();
+			changeAngleToPosition( event );
+		},
+		onDragMove: changeAngleToPosition,
+		onDragEnd: changeAngleToPosition,
+	} );
+	return	(
+		/* eslint-disable jsx-a11y/no-static-element-interactions */
+		<div
```

Comment:
You can't control it with the keyboard at all. It would be great to get some feedback from the accessibility experts on the best practices here, cc @tellthemachines and @enriquesanchez.

Reply:
Hi @gziolo, I added aria-hidden to the circular picker. It is hidden to screen reader users. We have a text field that allows setting the angle with the keyboard, the field also support arrows (up/down) to increase/decrease the angle. So I think we are covered we provide a UI for keyboard users and on for mouse users.

Reply:
I don't mean to block all the great work in this PR. We can revisit it later. I just wanted to make sure it's properly tested. 

By the way, I don't see `aria-hidden` used in this PR. What do I miss?

Reply:
Sorry I missed the part where I actually save the file. It is fixed now 👍